### PR TITLE
improve generating options docs, generating bins moduledocs

### PR DESCRIPTION
**Pull request by mat-hek**
should be merged after #282

### CODE REVIEW

File: lib/membrane/core/child.ex [hunk 1]
```diff
@@ -2,4 +2,50 @@ defmodule Membrane.Core.Child do
...
   @type state_t :: Membrane.Core.Element.State.t() | Membrane.Core.Bin.State.t()
+
+  @docs_order [
+    :membrane_options_moduledoc,
+    :membrane_pads_moduledoc,
+    :membrane_clock_moduledoc
+  ]
+
+  @spec generate_moduledoc(module, :element | :bin) :: Macro.t()
+  def generate_moduledoc(module, child) do
```
Comment (bblaszkow06) [inline lib/membrane/core/child.ex:13]: Maybe `child_type` would be more descriptive? Or something similar?

File: lib/membrane/core/child.ex [hunk 2]
```diff
@@ -2,4 +2,50 @@ defmodule Membrane.Core.Child do
...
+      case Module.get_attribute(module, :moduledoc, "") do
+        {_line, doc} -> doc
+        doc -> doc
+      end
+
+    if moduledoc != false do
+      moduledoc =
+        if String.trim(moduledoc) == "" do
+          quote do
+            unquote(moduledoc) <> "Membrane #{unquote(child)}.\n\n"
```
Comment (bblaszkow06) [inline lib/membrane/core/child.ex:30]: What is the purpose of concatenating whitespaces (`String.trim(moduledoc) == ""`) to that line?

File: lib/membrane/core/child/pads_specs.ex [hunk 3]
```diff
@@ -222,7 +222,7 @@ defmodule Membrane.Core.Child.PadsSpecs do
       config
       |> Enum.map(fn {k, v} ->
         {
-          k |> to_string() |> String.replace("_", "&nbsp;") |> String.capitalize(),
+          k |> to_string() |> String.replace("_", " ") |> String.capitalize(),
```
Comment (bblaszkow06) [inline lib/membrane/core/child/pads_specs.ex:225]: I'm pretty sure it was there for a reason, regular space may allow line break. Are you sure this change doesn't break anything?

Comment (mat-hek) [reply lib/membrane/core/child/pads_specs.ex:225]: `&nbsp;` just doesn't work there anymore, it's just displayed as is, like any html

**Review by bblaszkow06 — APPROVED** [review]

**Pull request merged**